### DISCUSSION TITLE
Tomb dependences

### DISCUSSION THREAD

**Discussion by hektr** [dyne/tomb#420] [General]
Tomb package in [AUR](https://aur.archlinux.org/packages/tomb/) does have dependencies `bc` and `inetutils`. Is it really necessary for tomb?

**Comment by roddhjav** [comment]
`inetutils` is required because Tomb needs `hostname`. `bc` is not required. I will update the PKGBUILD in the AUR. Thanks for the hint.

**Reply by hektr** [reply]
thanks :)
